lib/classes/Variables.test.js

```diff
@@ -638,19 +638,19 @@ describe('Variables', () => {
     it('should get undefined if non existing file and the second argument is true', () => {
       const serverless = new Serverless();
       const tmpDirPath = testUtils.getTmpDirPath();
-      const fileToTest = './config.yml';
-      const expectedFileName = path.join(tmpDirPath, fileToTest);
 
       serverless.config.update({ servicePath: tmpDirPath });
 
-      const realpathSync = sinon
-        .stub(fse, 'realpathSync').returns(expectedFileName);
+      const realpathSync = sinon.spy(fse, 'realpathSync');
+      const existsSync = sinon.spy(fse, 'existsSync');
 
-      return serverless.variables.getValueFromFile(`file(${fileToTest})`)
+      return serverless.variables.getValueFromFile('file(./non-existing.yml)')
         .then(valueToPopulate => {
-          expect(realpathSync.calledWithMatch(expectedFileName));
+          expect(realpathSync.calledOnce).to.be.equal(false);
```

Comment:
we should use 
```
expect(realpathSync).to.not.have.called;
expect(existsSync).to.have.been.calledOnce;
```
with `sinon-chai`. This leads to much better error messages in case of failures.

Reply:
Thanks @HyperBrain :+1: 
I already merged this PR :sweat_smile: I will create another PR contains this request.

Reply:
Umm, `no-unused-expressions` linting error appears while using syntax with `sinon-chai`.
I'm not sure a best way how to fix it.

Reply:
The most viable solution is, to declare `// eslint-disable no-unused-expression` at the top of the unit test file. I think @pmuens also did that in other locations. Using chai this is a standard pattern.